README.md

```diff
@@ -1,11 +1,12 @@
-# terraform-aws-alb-ecs-task [![](https://github.com/rhythmictech/terraform-aws-alb-ecs-task/workflows/pre-commit-check/badge.svg)](https://github.com/rhythmictech/terraform-aws-alb-ecs-task/actions) <a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=RhythmicTech" alt="follow on Twitter"></a>
+# terraform-aws-alb-ecs-task [![](https://github.com/rhythmictech/terraform-aws-alb-ecs-task/workflows/pre-commit-check/badge.svg)](https://github.com/rhythmictech/terraform-aws-alb-ecs-task/actions) <a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=twitter" alt="follow on Twitter"></a>
 Creates an ECS service, ECS task, ALB target group, ALB listener, and CloudWatch logging. Ignores updates to the task so deployments can continue via another pipeline.
 
 ## Example
 Here's what using the module will look like
 ```hcl
 module "example" {
-  source = "github.com/rhythmictech/terraform-aws-alb-ecs-task?ref=master"
+  source  = "rhythmictech/alb-ecs-task/aws"
+  version = "1.1.0"
```

Comment:
I don't feel too strongly but is there any reason to have any version ref at all in the example? It's just a thing that we'll let get out of date.

Reply:
🤷 because that's how it'll look in a non-example? They have badges for that btw ^.^

![GitHub release (latest by date)](https://img.shields.io/github/v/release/rhythmictech/terraform-aws-alb-ecs-task?logoColor=green)

https://shields.io/category/version